utils/results/conversion/convertor.go

```diff
@@ -0,0 +1,191 @@
+package conversion
+
+import (
+	"strings"
+
+	"github.com/jfrog/jfrog-cli-security/utils/formats"
+	"github.com/jfrog/jfrog-cli-security/utils/results"
+	"github.com/jfrog/jfrog-cli-security/utils/results/conversion/sarifparser"
+	"github.com/jfrog/jfrog-cli-security/utils/results/conversion/simplejsonparser"
+	"github.com/jfrog/jfrog-cli-security/utils/results/conversion/summaryparser"
+	"github.com/jfrog/jfrog-cli-security/utils/results/conversion/tableparser"
+	"github.com/jfrog/jfrog-cli-security/utils/techutils"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+	"github.com/owenrumney/go-sarif/v2/sarif"
+)
+
+type CommandResultsConvertor struct {
+	Params ResultConvertParams
+}
+
+type ResultConvertParams struct {
```

Comment:
In every interface implementation I see a Get function. Should we add it to the interface as well so if a new convertor will have to be created we will keep the unified form? I see the return value is different for each convertor so only if possible